README.md

```diff
@@ -3,22 +3,23 @@
 [![Lint Status](https://github.com/AMWA-TV/nmos-template/workflows/Lint/badge.svg)](https://github.com/AMWA-TV/nmos-template/actions?query=workflow%3ALint)
 [![Render Status](https://github.com/AMWA-TV/nmos-template/workflows/Render/badge.svg)](https://github.com/AMWA-TV/nmos-template/actions?query=workflow%3ARender)
 
-This repository holds the source for this Specification, part of the family of [Networked Media Open Specifications](https://specs.amwa.tv/nmos) from the [Advanced Media Workflow Association](https://amwa.tv)
+This repository holds the source for this Implementation Guide, part of the family of [Networked Media Open Specifications](https://specs.amwa.tv/nmos) from the [Advanced Media Workflow Association](https://amwa.tv)
```

Comment:
>part of the family of Networked Media Open Specifications

Kind of suggests its a spec.  Maybe something like:

>This repository holds the source for this Implementation Guide, _which is a guide to_ the Networked Media Open Specifications from the Advanced Media Workflow Association

Reply:
Wayward nephew? Touchy-feely uncle?

Reply:
Probably best to leave as 'part of the family' then :)